go.mod

```diff
@@ -120,9 +120,11 @@ require (
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.24.0 // indirect
 	golang.org/x/sync v0.1.0 // indirect
-	golang.org/x/sys v0.7.0
+	golang.org/x/sys v0.8.0
 	google.golang.org/genproto v0.0.0-20230403163135-c38d8f061ccd // indirect
 	google.golang.org/grpc v1.54.0 // indirect
 	google.golang.org/protobuf v1.30.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+replace github.com/minio/minio-go/v7 v7.0.52 => github.com/krisis/minio-go/v7 v7.0.0-20230524223141-ad61f9f521a9
```

Comment:
Will this be reverted once base PR of minio-go merged? I understand so.

Reply:
Yes, I will update this PR to use github.com/minio/minio-go/v7.

Reply:
Done